### ISSUE TITLE
New user interface using Textual would be nice

### ISSUE THREAD

**Issue by yudilee** [dipu-bd/lightnovel-crawler#1357]
Yesterday i see https://www.textualize.io/ has really nice user interface for command line app. Maybe this would be nice refresh to user interface

**Comment by UltraStudioLTD** [comment]
@dipu-bd,
I don't recommend to do it for now. Reason is that API is currently unstable.

**Comment by stacknoo** [comment]
Is the api stable now the user interface looks really nice

**Comment by dipu-bd** [comment]
We have the web app now. This is no longer required

**Comment by stacknoo** [comment]
Yeah i know but is the console bot interface going to get an upgrade

**Comment by dipu-bd** [comment]
Yes, it will get a major upgrade soon

**Comment by dipu-bd** [comment]
Closing as stale. This predates the rework that replaced the CLI-centred app with a server and web UI, and it has had no activity since. That is not a judgement on the idea — the ground it was written against simply is not there any more.

If you still want it against the current app, please reopen or file fresh with what it should look like now.
